rotkehlchen/chain/avalanche/manager.py

```diff
@@ -50,35 +45,20 @@ def get_latest_block_number(self) -> int:
         return self.w3.eth.block_number
 
     def get_avax_balance(self, account: ChecksumEvmAddress) -> FVal:
-        """Gets the balance of the given account in AVAX
-
-        May raise:
-        - RemoteError if Covalent is used and there is a problem querying it or
-        parsing its response
-        """
-        result = self.covalent.get_token_balances_address(account)
-        if result is None:
+        """Gets the balance of the given account in AVAX"""
+        balance = ZERO
+        try:
             balance = from_wei(FVal(self.w3.eth.get_balance(account)))
-        else:
-            balance = ZERO
-            for entry in result:
-                if entry.get('contract_ticker_symbol') == 'AVAX':
-                    balance = from_wei(FVal(entry.get('balance', 0)))
-                    break
-            if balance == ZERO:
-                balance = from_wei(FVal(self.w3.eth.get_balance(account)))
-        return FVal(balance)
+        except Web3Exception as e:
+            log.error(f'Failed to connect to AVAX rpc node. {e}')
```

Comment:
no need to catch this error here, right? Perhaps just say that this can raise Web3Exception and then handle it in the only place where this function is called.